assembl/templates/index_react.jinja2

```diff
@@ -2,7 +2,9 @@
 {% block page_title%}Assembl{% endblock%}
 
 {% block main_css %}
-	{# Override the main css. #}
+  {# Override the main CSS. #}
+  <link type="text/css" rel="stylesheet" href="/static2/build/bundle.css" />
```

Comment:
Yes, "bundle" is the name of the main "entry" of webpack.config.js (see file). So webpack produces a bundle.js as well as a bundle.css, which contains CSS mentioned in all files which have been agregated in bundle.js.